Library/Homebrew/formula.rb

```diff
@@ -2932,11 +2932,20 @@ def pour_bottle?(only_if: nil, &block)
       when :clt_installed
         lambda do |_|
           on_macos do
-            T.cast(self, PourBottleCheck).reason(+<<~EOS)
-              The bottle needs the Apple Command Line Tools to be installed.
-                You can install them, if desired, with:
-                  xcode-select --install
-            EOS
+            if MacOS::Xcode.installed?
+              T.cast(self, PourBottleCheck).reason(+<<~EOS)
+                This bottle requires the standalone Apple Command Line Tools to be installed.
+                You have an Xcode installation, and the Xcode Command Line Tools are not sufficient.
+                  You can install the Apple Command Line Tools in addition to Xcode, if desired, with:
+                    xcode-select --install
```

Comment:
```suggestion
                  You can install the Apple Command Line Tools in addition to Xcode, if desired, with:
                    xcode-select --install
```

I'd suggest using `MacOS::CLT.installation_instructions` both here and below.